src/api.jl

```diff
@@ -1,7 +1,11 @@
-export construct_univariate_relaxation!, construct_bilinear_relaxation!
+export construct_univariate_relaxation!, 
+    refine_partition!,
+    construct_bilinear_relaxation!,
+    construct_univariate_on_off_relaxation!
 
 """
-    construct_univariate_relaxation!(m,f,x,y,x_partition;f_dash=x->ForwardDiff.derivative(f,x),error_tolerance=NaN64,length_tolerance=1e-6,derivative_tolerance=1e-6,num_additional_partitions=0)
+    construct_univariate_relaxation!(m,f,x,y,x_partition;f_dash=x->ForwardDiff.derivative(f,x),error_tolerance=NaN64,length_tolerance=1e-6,derivative_tolerance=1e-6,num_additional_partitions=0,
+    constraint_pre_base_name="",variable_pre_base_name="",use_formulation_info=FormulationInfo())
```

Comment:
Looks like the correct parameter name is `formulation_info` and not `use_formulation_info`.